Client.xcodeproj/project.pbxproj

```diff
@@ -15250,28 +16237,98 @@
 			name = Today.strings;
 			sourceTree = "<group>";
 		};
-		2C6188DB2B7A8A22006B70D7 /* Ecosia.strings */ = {
+		2CD263C92CFDC76800A040A7 /* Ecosia.strings */ = {
+			isa = PBXVariantGroup;
+			children = (
+				2CD263C82CFDC76800A040A7 /* de */,
+			);
+			name = Ecosia.strings;
+			sourceTree = "<group>";
+		};
+		2CD263CB2CFDC76800A040A7 /* Ecosia.strings */ = {
+			isa = PBXVariantGroup;
+			children = (
+				2CD263CA2CFDC76800A040A7 /* en */,
+			);
+			name = Ecosia.strings;
+			sourceTree = "<group>";
+		};
+		2CD263CD2CFDC76800A040A7 /* Ecosia.strings */ = {
+			isa = PBXVariantGroup;
+			children = (
+				2CD263CC2CFDC76800A040A7 /* es */,
+			);
+			name = Ecosia.strings;
+			sourceTree = "<group>";
+		};
+		2CD263CF2CFDC76800A040A7 /* Ecosia.strings */ = {
+			isa = PBXVariantGroup;
+			children = (
+				2CD263CE2CFDC76800A040A7 /* it */,
+			);
+			name = Ecosia.strings;
+			sourceTree = "<group>";
+		};
+		2CD263D12CFDC76800A040A7 /* Ecosia.strings */ = {
+			isa = PBXVariantGroup;
+			children = (
+				2CD263D02CFDC76800A040A7 /* fr */,
+			);
+			name = Ecosia.strings;
+			sourceTree = "<group>";
+		};
+		2CD263D32CFDC76800A040A7 /* Ecosia.strings */ = {
 			isa = PBXVariantGroup;
 			children = (
-				2C6188DC2B7A8A22006B70D7 /* de */,
-				2C6188DF2B7A8A22006B70D7 /* en */,
-				2C6188E12B7A8A22006B70D7 /* es */,
-				2C6188E32B7A8A22006B70D7 /* it */,
-				2C6188E92B7A8A22006B70D7 /* fr */,
-				2C6188EB2B7A8A22006B70D7 /* nl */,
+				2CD263D22CFDC76800A040A7 /* nl */,
 			);
 			name = Ecosia.strings;
 			sourceTree = "<group>";
 		};
```

Comment:
This feel a bit strange as it's now considering each language as a separate strings file instead of just variations of the same file like we are used to 🤔 I don't think this will be a problem for localisation but might be a sign something is not right? Do you have an idea why that is?

![Screenshot 2024-12-18 at 12 01 02](https://github.com/user-attachments/assets/18cd8947-c151-49e6-9c17-274340ec8f6a)

Reply:
That's purely visual. I double-checked the files and the way they are added is the same.
We can think about it or try to read them but SnapshotsTests are also passing.
That's a sign everything is good localization-wise. Also double-checked myself updating the language.